src/modules/Speech_Recognition/Whisper.py

```diff
@@ -80,6 +100,10 @@ def transcribe_with_whisper(
         )
         sys.exit(1)
 
+    # convert any numbers to words so align will have timing
+    for obj in result["segments"]:
```

Comment:
It only works for English songs. With other languages you would have english words.